packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -53,13 +53,18 @@ export function disableCLITelemetry() {
 export async function isCLITelemetryAllowed(
   projectPath: string
 ): Promise<boolean> {
-  // if (isCLIInstalled()) {
-  //   const forceConfig = await new ForceConfigGet().getConfig(
-  //     projectPath,
-  //     SFDX_CONFIG_DISABLE_TELEMETRY
-  //   );
-  //   const disabledConfig = forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) || '';
-  //   return disabledConfig !== 'true';
-  // }
+  if (isCLIInstalled()) {
```

Comment:
What happens if we don't check for the CLI being present ? I'm asking cause this will add a second `isCLIInstalled()` check to the activation of core and that call is not the most efficient

Reply:
@lcampos Based on my testing the settings check will hang indefinitely if the cli is not installed. This case is difficult to trap without more extensive changes.

Reply:
Ok, let's keep it as is and we can have a follow up story to address the perf hit.